astropy/coordinates/name_resolve.py

```diff
@@ -49,6 +49,10 @@ class sesame_database(ScienceState):
     using the name resolve mechanism in the coordinates
     subpackage. Default is to search all databases, but this can be
     'all', 'simbad', 'ned', or 'vizier'.
+
+    If "all" is selected, the answer is first requested from SIMBAD,
```

Comment:
This is a total nitpick, and I only ask because the changelog entry request means a new push anyway: since above the strings are with single quotes, could you use 'all' here too?